src/alfanous-desktop/Gui.py

```diff
@@ -824,16 +824,16 @@ def help( self ):
         self.o_results.setText( html )
     
     def share_an_idea(self):
-        webbrowser.open('http://feedback.alfanous.org/response/add/idea/?text=[DESKTOP]')
+        webbrowser.open('https://github.com/Alfanous-team/alfanous/issues/')
     
     def ask_a_question(self):
-        webbrowser.open('http://feedback.alfanous.org/response/add/question/?text=[DESKTOP]')
+        webbrowser.open('https://github.com/Alfanous-team/alfanous/issues/')
```

Comment:
for desktop, its better to keep it and make a redirection in the server... it will take long for this to be released

Reply:
Well, redirection for already released packages.  But why you want to keep it for future package releases. Are you expecting another change for (question-tracker)?

Reply:
not for now, but possibly

Reply:
Ok @assem-ch rolled back for GUI